chia/consensus/multiprocess_validation.py

```diff
@@ -274,34 +263,25 @@ async def pre_validate_blocks_multiprocessing(
         except ValueError:
             return [PreValidationResult(uint16(Err.INVALID_SUB_EPOCH_SUMMARY.value), None, None, False, uint32(0))]
 
-        if block_rec.sub_epoch_summary_included is not None:
-            if wp_summaries is not None:
-                idx = int(block.height / constants.SUB_EPOCH_BLOCKS) - 1
-                next_ses = wp_summaries[idx]
-                if not block_rec.sub_epoch_summary_included.get_hash() == next_ses.get_hash():
-                    log.error("sub_epoch_summary does not match wp sub_epoch_summary list")
-                    return [
-                        PreValidationResult(uint16(Err.INVALID_SUB_EPOCH_SUMMARY.value), None, None, False, uint32(0))
-                    ]
+        if block_rec.sub_epoch_summary_included is not None and wp_summaries is not None:
+            idx = int(block.height / constants.SUB_EPOCH_BLOCKS) - 1
+            next_ses = wp_summaries[idx]
```

Comment:
```suggestion
             next_ses = wp_summaries[ int(block.height / constants.SUB_EPOCH_BLOCKS) - 1] 
```


and removing line 267